lib/internal/child_process.js

```diff
@@ -298,7 +298,7 @@ function flushStdio(subprocess) {
     // which data can be read from a stream, e.g. being consumed on the
     // native layer directly as a StreamBase.
     if (!stream || !stream.readable ||
-        stream._readableState.readableListening ||
+        stream.readableListening ||
```

Comment:
Do we get readable listening without it's state?

Reply:
I don’t understand your question.

Reply:
@patilharshal16, hopefully the updated explanation of the property in [stream.md](doc/api/stream.md) clarifies it for you.

Reply:
Isn't `stream.listenerCount('readable') > 0` enough here?